FoxDot/lib/Settings/__init__.py

```diff
@@ -122,6 +122,20 @@ def GET_TUTORIAL_FILES():
 
     FOXDOT_SND = os.path.realpath(conf.SAMPLES_DIR)
 
+# Recreate info file from template
+if os.path.isfile(FOXDOT_INFO_FILE):
+    os.remove(FOXDOT_INFO_FILE)
```

Comment:
Might be better to move this into the try/except before you write the file? Just in case - for whatever reason - the template file doesn't exist and then there is no info file on start up

Reply:
Indeed that makes sense, already pushed the update. Thanks!